app/services/plans/update_service.rb

```diff
@@ -197,5 +199,16 @@ def discard_charge!(charge)
 
       Invoice.where(id: draft_invoice_ids).update_all(ready_to_be_refreshed: true) # rubocop:disable Rails/SkipsModelValidations
     end
+
+    # NOTE: We should remove pending subscriptions
+    #       if plan has been downgraded but amount cents became less than downgraded value. This pending subscription
+    #       is not relevant in this case and downgrade should be ignored
+    def process_downgraded_subscriptions
+      return unless plan.subscriptions.active.exists?
+
+      Subscription.where(previous_subscription_id: plan.subscriptions.active, status: :pending).each do |sub|
```

Comment:
Rails/FindEach: Use `find_each` instead of `each`. (https://rails.rubystyle.guide#find-each)